src/cdk/drag-drop/drag-ref.ts

```diff
@@ -88,6 +88,12 @@ export interface Point {
   y: number;
 }
 
+/** Inline styles to be set as `!important` while dragging. */
+const dragImportantProperties = new Set([
+  // Needs to be important, because some `mat-table` sets `position: sticky !important`. See #22781.
+  'position'
+]);
+
```

Comment:
Would problematic would it be to alternatively remove the `!important` from `mat-table`? My inclination would be to avoid an `!important` arms race by avoiding it completely.

Reply:
The problem with removing the `!important` is that our sticky header selector currently has really low specificity and can be overwritten by styles that have been encapsulated by Angular. See https://github.com/angular/components/issues/20558.